server/etcdserver/cindex/cindex.go

```diff
@@ -23,9 +23,9 @@ import (
 )
 
 var (
-	metaBucketName = []byte("meta")
+	MetaBucketName = []byte("meta")
 
-	consistentIndexKeyName = []byte("consistent_index")
+	ConsistentIndexKeyName = []byte("consistent_index")
```

Comment:
What's the use case for making this public? If we need, can we document this saying this belongs to meta bucket?

Reply:
I'm trying to get rid of duplicated independent constants. In particular this one is used in: server/mvcc/kvstore.go for list of ignored keys when 'hash' is being computed, so no easy way to merge it into cindex package. 

Parted port of my PR to this one, to make it consistent: 9fb0aa3